packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
+
+#### Drill-down navigation
+
+Drill-down menus replace the current navigation menu with the next set of child items in the page hierarchy when the user clicks on a parent item to drill down. A back link is provided at the top of the menu to return to the parent level.
+
+<img src="./img/drill-down.png" alt="Example of drill-down navigation" width="916"/>
+
+Using drill-down navigation, only the navigation items for the current page and its siblings will be visible at any given time. This pattern is very mobile-friendly as it minimizes the need for scrolling to see the entire menu. However, it should be avoided if you expect users to frequently move between levels.
+
+### Combining vertical navigation patterns
+
+PatternFly's vertical menus are designed to be composable. This means that you can combine different patterns within the same navigation system to address the needs of users working at different levels in the page hierarchy.
+
+When combining different types of menus, here are some guidelines:
+
+* Use a consistent approach to represent items at the same level in the page hierarchy. For example, if drill-down is used from level 1 - 2, all items with children at that level should display the same behavior. Don’t mix fly-outs, drill-downs, and expansion at the same level.
+
+* Consider how important or likely it is for users to move between levels and/or across to items at the same level but in different branches of the hierarchy. Drill-downs are most effective when users will spend most of their time moving between pages with the same parent node. Therefore, drilling into a lower level in the page hierarchy and then employing expansions or fly-outs to navigate between pages at that level can be an effective approach.
+
+Here are some examples of hybrid navigation patterns that you may find useful.
+
+#### Fly-outs with tertiary drill-down
+By using fly-outs to expose secondary navigation items and then drilling into third level items, you can keep the current page and its siblings visible as the user works. This is a good pattern to use when you expect users to spend most of their time working within the same section of the application.
+
+<img src="./img/flyout-to-drill-down.png" alt="Example of composable navigation with fly-out and drill-down menus" width="1143"/>
+
+#### Two-level expansion with drill-downs
+Similarly, you can use expanded menus to expose the first two or three levels of page hierarchy and then drill-down to the lowest level. This pattern is preferred to using fly-outs with a drill-down when there are fewer secondary level items and/or you require a mobile friendly solution.
+
+<img src="./img/expansion-to-drill-down.png" alt="Example of composable navigation with expansion and drill-down menus" width="928"/>
+
+#### Drill-down with tertiary expansion
```

Comment:
Question just for my knowledge - in this case, if they click on an item in the drill-down state, does the nav stay in the drill down state, or does it go back to the main state?
I'm assuming it remains in the state that surfaces the nav item they are on?

Reply:
Yes